base/special/trig.jl

```diff
@@ -200,6 +200,25 @@ ACOS_CORRECT_LOWWORD(::Type{Float64}, x) = reinterpret(Float64, (reinterpret(UIn
 
 @noinline acos_domain_error(x) = throw(DomainError(x, "acos(x) not defined for |x|>1"))
 function acos(x::T) where T <: Union{Float32, Float64}
+    # Method :
```

Comment:
I'm confused. Couldn't one infer the method explained here simply by reading the code? Perhaps explanatory comments could go inline wherever something may be unclear rather than trying to match function logic to the description at the top? IMO anyway.

Reply:
I really like these clear comments. Inferring the method by reading the code for very low level stuff is a hard inverse problem.